ax12:
  max_angle_bytes: 0x3FF
  max_angle_degrees: 300
mx64:
  max_angle_bytes: 0xFFF
  max_angle_degrees: 360
mx28:
  max_angle_bytes: 0xFFF
  max_angle_degrees: 360
xl430:
  max_angle_bytes: 0xFFF
  max_angle_degrees: 360
xm430:
  max_angle_bytes: 0xFFF
  max_angle_degrees: 360
